QModManager/Dialog.cs

```diff
@@ -6,7 +6,6 @@
 using System.Diagnostics;
```

Comment:
This class could have a better name and could be moved someplace else, maybe in `Utility`?

Reply:
Utility seems like the perfect place for this. 👍